db/migrate/20160927071039_create_notification_types.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNotificationTypes < ActiveRecord::Migration
+  def change
+    create_table :notification_types do |t|
+      t.string :name, index: true, unique: true
```

Comment:
should we add a name and a group? esp if its going to become a blueprint i would think:
name - provisioning error, group: host or provisioning